tests/test_psf.py

```diff
@@ -209,14 +206,15 @@ def setupDeterminer(
         self.starSelector = starSelectorClass(config=starSelectorConfig)
 
         makePsfCandidatesConfig = measAlg.MakePsfCandidatesTask.ConfigClass()
-        if stampSize is not None:
-            makePsfCandidatesConfig.kernelSize = stampSize
+        makePsfCandidatesConfig.kernelSize = stampSize
```

Comment:
How do you want to do this now, considering we need larger kernelSize for these tests?

Reply:
Before I could respond to that, I think one question from Slack still remains. Should `modelSize` be configurable in addition to `stampSize`, or should we derive it from `stampSize` as `1.414*stampSize`, rounded up to an odd integer? I see that having it smaller than `1.414*stampSize` can truncate the edges, but I am not sure if there's an advantage to having it larger than required. 

> Should we add more test cases?

If we go ahead with this being configurable, we need to add tests with larger values of `modelSize` to ensure that 1) piff can handle it without crashing 2) we get consistent (or even identical) values for the core of the PSF, if that's that expectation.

Reply:
This PR adds a check that, when using sky coords, the `stampSize > sqrt(2) * modelSize`

Reply:
Ah, I had read that inequality the opposite way in my first pass of the PR. The current code would allow running this with `modelSize = 25` and `stampSize=25`, even when `useCoordinates = 'sky'` right? I can't imagine a use case where that would be desirable. Is there any?

Reply:
Unfortunate typos in the above comment that caused much of the confusion. I meant to ask about `modelSize=15` and `stampSize=35` when `useCoordinate = 'sky'`.  We should include tests that run with `modelSize=15` but `stampSize=25`, `stampSize=35` and `stampSize=45` to see if they give consistent results (these configs are not prohibited by `_check_stamp_size` function). The reason I think it's not trivial is that `drawSize` directly depends on `stampSize` and I'm not sure what behavior to expect when piff tries to render a PSF image much larger than its internal model size.

Reply:
Running with `modelSize=15, stampSize=25` causes the tests to fail due to decreased accuracy of the model

Reply:
Ah, is that because the `modelSize` of 15 is too small for the PSF? Perhaps `modelSize=25` and `stampSize=45`?

Reply:
By consistent results, do you mean they pass the tests or something more specific?

Reply:
Passing the existing tests at the minimum, and that should be sufficient to accept this PR. It'd be nice if you could include another test that compares the inner 10x10 PSF image when `stampSize=35` and `stampSize=45` and find that they are identical, up to a scaling factor perhaps that accounts for flux normalization.

Reply:
Do we expect those to be identical?

I can imagine the iteration proceeding differently for the different stamp sizes.

Reply:
OK, I wasn't sure about whether that's a reasonable expectation in the first place. Happy to skip testing that if we shouldn't be expecting them to be the same.